diff -r -U 3 minecraft\net\minecraft\src\CodecMus.java minecraft_patched\net\minecraft\src\CodecMus.java
--- minecraft\net\minecraft\src\CodecMus.java	Tue Jul 05 00:59:58 2011
+++ minecraft_patched\net\minecraft\src\CodecMus.java	Tue Jul 05 01:03:31 2011
@@ -1,5 +1,6 @@
 package net.minecraft.src;
 
+import java.io.IOException;
 import java.io.InputStream;
 import net.minecraft.src.MusInputStream;
 import paulscode.sound.codecs.CodecJOrbis;
@@ -7,6 +8,10 @@
 public class CodecMus extends CodecJOrbis {
 
    protected InputStream openInputStream() {
-      return new MusInputStream(this, this.url, this.urlConnection.getInputStream());
+      try {
+         return new MusInputStream(this, this.url, this.urlConnection.getInputStream());
+      } catch (IOException e) {
+         return null;
+      }
    }
 }
diff -r -U 3 minecraft\net\minecraft\src\ItemMap.java minecraft_patched\net\minecraft\src\ItemMap.java
--- minecraft\net\minecraft\src\ItemMap.java	Tue Jul 05 01:00:10 2011
+++ minecraft_patched\net\minecraft\src\ItemMap.java	Tue Jul 05 01:02:12 2011
@@ -19,7 +19,7 @@
    }
 
    public static MapData func_28013_a(short var0, World var1) {
-      "map_" + var0;
+//      "map_" + var0;
       MapData var3 = (MapData)var1.func_28103_a(MapData.class, "map_" + var0);
       if(var3 == null) {
          int var4 = var1.func_28101_b("map");
@@ -32,7 +32,7 @@
    }
 
    public MapData func_28012_a(ItemStack var1, World var2) {
-      "map_" + var1.func_21181_i();
+//      "map_" + var1.func_21181_i();
       MapData var4 = (MapData)var2.func_28103_a(MapData.class, "map_" + var1.func_21181_i());
       if(var4 == null) {
          var1.func_28156_b(var2.func_28101_b("map"));
diff -r -U 3 minecraft\net\minecraft\src\MusInputStream.java minecraft_patched\net\minecraft\src\MusInputStream.java
--- minecraft\net\minecraft\src\MusInputStream.java	Tue Jul 05 01:00:02 2011
+++ minecraft_patched\net\minecraft\src\MusInputStream.java	Tue Jul 05 01:04:05 2011
@@ -1,5 +1,6 @@
 package net.minecraft.src;
 
+import java.io.IOException;
 import java.io.InputStream;
 import java.net.URL;
 import net.minecraft.src.CodecMus;
@@ -28,7 +29,11 @@
    }
 
    public int read(byte[] var1, int var2, int var3) {
-      var3 = this.field_1515_d.read(var1, var2, var3);
+      try {
+         var3 = this.field_1515_d.read(var1, var2, var3);
+      } catch (IOException e) {
+         return 0;
+      }
 
       for(int var4 = 0; var4 < var3; ++var4) {
          byte var5 = var1[var2 + var4] = (byte)(var1[var2 + var4] ^ this.field_1516_c >> 8);
